batcher/aligned-batcher/src/lib.rs

```diff
@@ -798,8 +1132,7 @@ impl Batcher {
             batch_data_pointer.clone(),
             padded_leaves.clone(),
             signatures.clone(),
-            gas_for_aggregator,
-            gas_per_proof,
+            fee_params.clone(),
             &self.payment_service,
         )
         .await
```

Comment:
If this transaction reverts, it is not informed to the client

Reply:
it is being informed, client is not logging it, not related to this pr